airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml

```diff
@@ -1,91 +1,79 @@
-        status: "failed"
+      - config_path: secrets/config_oauth.json
+        status: succeed
+      - config_path: integration_tests/invalid_config.json
+        status: failed
+      - config_path: integration_tests/invalid_config_oauth.json
+        status: failed
+      - config_path: integration_tests/invalid_config_wrong_title.json
+        status: failed
   discovery:
     tests:
-      - config_path: "secrets/config.json"
-        backward_compatibility_tests_config:
-          # Quote stream is deleted so it is not backward compatible with previous discovery
-          disable_for_version: "0.2.3"
+      - config_path: secrets/config_oauth.json
   basic_read:
     tests:
-      - config_path: "secrets/config.json"
-        timeout_seconds: 600
-        configured_catalog_path: "sample_files/basic_read_oauth_catalog.json"
+      - config_path: secrets/config_oauth.json
         empty_streams:
-          - name: "form_submissions"
-            bypass_reason: "unable to populate"
-          - name: "form_submissions"
-            bypass_reason: "unable to populate"
-          - name: "ticket_pipelines"
-            bypass_reason: "unable to populate"
-          - name: "engagements_meetings"
-            bypass_reason: "unable to populate"
-          - name: "engagements_emails"
-            bypass_reason: "unable to populate"
-          - name: "engagements"
-            bypass_reason: "unable to populate"
-          - name: "engagements_calls"
-            bypass_reason: "unable to populate"
-          - name: "quotes"
-            bypass_reason: "unable to populate"
-        # This test commented out, since it produces errors during active testing
-        # expect_records:
-        #   path: "integration_tests/expected_records.jsonl"
-  incremental:
-    tests:
-      - config_path: "secrets/config.json"
-        configured_catalog_path: "sample_files/incremental_catalog.json"
-        future_state:
-          future_state_path: "integration_tests/abnormal_state.json"
+        - name: companies
+          bypass_reason: unable to populate
+        - name: deals
+          bypass_reason: unable to populate
+        - name: property_history
+          bypass_reason: unable to populate
+        - name: tickets
+          bypass_reason: unable to populate
+        - name: form_submissions
+          bypass_reason: unable to populate
+        - name: ticket_pipelines
+          bypass_reason: unable to populate
+        - name: engagements_meetings
+          bypass_reason: unable to populate
+        - name: engagements_emails
+          bypass_reason: unable to populate
+        - name: engagements
+          bypass_reason: unable to populate
+        - name: engagements_calls
+          bypass_reason: unable to populate
+        - name: quotes
+          bypass_reason: unable to populate
+        - name: contacts_list_memberships
+          bypass_reason: unable to populate
```

Comment:
Since you have a contact, it should be possible to add them to a list

Reply:
added